.browserslistrc

```diff
@@ -0,0 +1,2 @@
+> 1%
```

Comment:
Do you know what this file is?

Reply:
this file came from the vue cli, it's let you define the browsers you support
and because we only using electron that uses chrome we really don't need this file